src/server/controllers/users.controller.js

```diff
@@ -13,8 +13,8 @@ let mockData = {
 };
 
 export function saveTest(req, res, next) {
-  let user = new user(mockData);
-  user.save((err) => {
+  let userSample = new user(mockData);
```

Comment:
I'm a bit confused. The contents in this file seem to be primarily for testing. Shouldn't it be in src/test directory then?